docs/sql-guide/functions/function-totimestamp.md

```diff
@@ -0,0 +1,78 @@
+---
+title: TOTIMESTAMP()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+
+# TOTIMESTAMP() function
+
+The `TOTIMESTAMP()` function converts the given integer value to a timestamp and returns it. It uses the [unix epoch](https://www.unixtutorial.org/unix-epoch/) as the base and calculates date-time equivalent of the integer.
+
+## Syntax
+
+```
+TOTIMESTAMP(int_exp, timeunit)
+```
+
+## Arguments
+
+
+| Argument | Description | Data type |
+|---|---|---|
+| `int_exp` | Required `int` value or expression to be converted to a `timestamp`. | `int` |
```

Comment:
| `int_val` | integer | Integer value to be converted to a timestamp | Optional |
| `expr` |   | expression used to derive an integer value to be converted to a timestamp | Optional |